fern/api/generators.yml

```diff
@@ -3,7 +3,7 @@ release:
     version: 0.0.209
     publishing:
       npm:
-        package-name: "@fern-api/raven" # TODO: should be published under Raven's NPM org
-        token: ${FERN_NPM_TOKEN} # TODO: should use Raven's NPM Token
+        package-name: "@ravenappdev/raven"
```

Comment:
https://www.npmjs.com/package/@ravenapp/raven-inapp-angular

will be using this organization to publish npm packages